### ISSUE TITLE
Error: ModuleNotFoundError: No module named 'cmdkit.contrib'

### ISSUE THREAD

**Issue by asarmakeeva** [GWU-CFD/FlashKit#71]
When running `flashkit create xdmf —auto` command in the case directory I getting `ModuleNotFoundError: No module named 'cmdkit.contrib'`

I am using python3.8 and FlashX

**Comment by akashdhruv** [comment]
Can you try installing `CmdKit` from this SHA

https://github.com/alentner/CmdKit/tree/1805df2bc174059ba4aced06eab8e92d8e5391d6

And try again? 

I would remove existing installs of `CmdKit` and `FlashKit`,  reinstall `CmdKit` it from this SHA using `python3 setup.py install` and then install `FlashKit` from here https://github.com/akashdhruv/FlashKit

**Comment by akashdhruv** [comment]
I think you maybe get an error from `importlib` saying `NoneType object as no attribute startswith` but I would still give it a try

**Comment by asarmakeeva** [comment]
I followed your instructions and run from the SHA, but it didn't help, I got same error

**Comment by glentner** [comment]
How do you have flashkit installed? In a virtual environment or just locally? You could be picking up an old copy of cmdkit from a different site despite the one you just installed. 

If you run a Python console, and import cmdkit, what is the location it gives you if you just type "cmdkit" into the console?

**Comment by asarmakeeva** [comment]
yeah, i installed flashkit locally, i piking up cmdkit from `'/home/username/CmdKit/cmdkit/__init__.py'>`

**Comment by akashdhruv** [comment]
This line needs to be removed to resolve the error

https://github.com/GWU-CFD/FlashKit/blob/97cb15f1aaaebb21ba9044e2197f85baba5b2f44/src/flashkit/core/configure.py#L17
